### PR TITLE
[BUGFIX][DEV-1187] Change required label on custom control

### PR DESCRIPTION
**Pull request by subjacked**

### CODE REVIEW

**Review by andreasfernandez — COMMENTED** [review]
It's not directly related to this change, but can we make the tick having `opacity: 1`?

image

**Comment by subjacked** [comment]
@andreasfernandez That would be no problem. But I think someone did it for some reason.

**Comment by andreasfernandez** [comment]
> @andreasfernandez That would be no problem. But I think someone did it for some reason.

The affecting CSS rule is the following one, this clearly looks like a side-effect:
```css
label.required:after {
    content: " *";
    opacity: .5;
}
```

**Comment by subjacked** [comment]
@andreasfernandez Sorry, you meant the tick, I was at the asterisk. I'll fix that.

File: assets/css/modules/_form.scss [hunk 1]
```diff
@@ -67,9 +67,12 @@ label {
...
     margin-left: 1rem;
 }
-.checkbox-custom.custom-control-label.required:after {
-    position: static;
-    display: inline;
+.custom-control-label .asterisk {
+    opacity: .5;
+}
+.custom-control-label.required:after {
+    content: '';
```
Comment (andreasfernandez) [inline assets/css/modules/_form.scss:74]: Do we still need this whole CSS rule since you've moved the asterisk into a separate element?

Comment (subjacked) [reply assets/css/modules/_form.scss:74]: @andreasfernandez Yes, to override the global .required:after

Comment (andreasfernandez) [reply assets/css/modules/_form.scss:74]: I see, the CSS still contains the `content: '*'` stuff.

**Review by andreasfernandez — APPROVED** [review]

**Pull request merged**